pkgs/development/python-modules/django_tagging/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchurl, django }:
+
+buildPythonPackage rec {
+  name = "django-tagging-0.4.5";
+
+  src = fetchurl {
```

Comment:
do you also want to convert instances of `fetchurl` to `fetchPypi` or do you want to keep this out of scope of pull request?